### PR TITLE
[AAP-6385] fix ansible dependencies for install-rulebook role

### PR DESCRIPTION
**Pull request by Alex-Izquierdo**
Fixes AAP-6385
Ansible and ansible-runner are indirect dependencies and must be managed in the system regardless of ansible-rulebook. The playbook started to install them by convenience. But since can be installed in a system scope, the playbook can fail trying to update them, so to avoid errors, better to move as "present" if those packages were previously installed by the user.

### CODE REVIEW

**Review by ttuffin — APPROVED** [review]

File: .github/workflows/integration-tests.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
-          python-version: "3.8"
+          python-version: "3.9"
```
Comment (konstruktoid) [inline .github/workflows/integration-tests.yml:19]: `python-version: "3.8"` is still okay? 
https://github.com/ansible/ansible-rulebook/commit/79892c2d1c168de6f9ae1842fcb940606086a0b4

Comment (Alex-Izquierdo) [reply .github/workflows/integration-tests.yml]: yeah, unmade. thanks @konstruktoid

File: roles/install_ansible_rulebook/tasks/install_fedora.yml [hunk 2]
```diff
@@ -32,15 +32,15 @@
...
+- name: Ensure ansible dependencies
   ansible.builtin.pip:
     name:
       - ansible
       - ansible-runner
-      - wheel
-    state: latest
+    state: present
 
 - name: install ansible-rulebook
```
Comment (ttuffin) [inline roles/install_ansible_rulebook/tasks/install_fedora.yml:45]:
```suggestion
- name: Install ansible-rulebook
```

Comment (ttuffin) [reply roles/install_ansible_rulebook/tasks/install_fedora.yml:45]: Can you capitalize the name on L53 as well?

Comment (Alex-Izquierdo) [reply roles/install_ansible_rulebook/tasks/install_fedora.yml:45]: Can you create a PR to fix all of those issues?

**Review by ttuffin — DISMISSED** [review]

**Review by bzwei — DISMISSED** [review]

**Review by ttuffin — APPROVED** [review]

**Review by ddonahue007 — APPROVED** [review]
lgtm

**Pull request merged**